qemu/tests/cfg/hotplug_unplug_with_viommu.cfg

```diff
@@ -0,0 +1,21 @@
+- hotplug_unplug_with_viommu:
+    virt_test_type = qemu
+    type = nic_hotplug
+    required_qemu= [7.0.0,)
+    only q35
```

Comment:
Need to change.

Reply:
virtio iommu should only be supported on the Q35 platform, isn't it? Please correct me if I'm wrong. Thanks a lot.

Reply:
https://gitlab.com/qemu-project/qemu/-/blob/master/softmmu/qdev-monitor.c#L59-64
https://gitlab.com/qemu-project/qemu/-/blob/master/softmmu/qdev-monitor.c#L92
https://gitlab.com/libvirt/libvirt/-/blob/master/src/qemu/qemu_validate.c#L4846-4853

Reply:
Ok, thanks for your help. But I'm not familiar with ARM, how should I specify it?

Reply:
Hello @PaulYuuu Can `only aarch64 x86_64` meet the requirements?